### ISSUE TITLE
Elevated circulating 4-Hydroxyphenylacetic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6427]
**Preferred term label:** Elevated circulating 4-Hydroxyphenylacetic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 4-Hydroxyphenylacetic acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating monocarboxylic acid cocentration HP:0010996

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
